content/index.fr.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Vue d'ensemble des standards de l'accessibilité"
+permalink: /fundamentals/fr
+ref: /fundamentals/
+lang: fr
+last_updated: 2020-12-28
+translators:
+- name : "Sofia Ahmed"
+contributors:
+- name : "Sandra Velarde Gonzalez (ETNIC)"
+github:
+  repository: w3c/wai-fundamentals-overview
+  path: content/index.fr.md
+footer: > # Text in footer in HTML
+  <p><strong>Date :</strong> Mise à jour : 19 août 2020.</p>
+  <p><strong>Rédactrice :</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Développé par le Groupe de travail Éducation et Promotion (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+
+---
+
+Les ressources suivantes contiennent des introductions à l'accessibilité :
+
+[[Introduction à l'accessibilité Web]](/fundamentals/accessibility-intro/)
+:   Introduit à l'accessibilité Web et renvoie à des ressources supplémentaires.
+    -   [[Vidéo : introduction à l'accessibilité Web et aux standards du W3C (4 minutes)]](/videos/standards-and-benefits/) .
+
+[[Éléments essentiels de l'accessibilité Web]](/fundamentals/components/)
```

Comment:
Title of the page: "Accessibility Fundamentals Overview"
Translation: "Vue d'ensemble des standards de l'accessibilité"

not sure if the translation reflects the page accurately. The page presents the "fundamentals" overview rather than specifically the "standards" as we talk also about the overall coverage of accessibility, i.e. the principles, the people etc and not only standards.

Reply:
Reply from Remi:

I concur. For other overview pages (Planning & Policies, Teach & Advocate, Design & Develop), I opted for the following construction : "[Topic] - Vue d'ensemble". If you also think it is a good approach, I suggest using it here for consistency.

Reply:
@remibetin
Would this work:  "Accessibilité - Vue d'ensemble" or "Aperçu des principes de base de l'accessibilité" or "Aperçu des principes fondamentaux de l'accessibilité"

Reply:
@vmmiller
My personal preference : "Principes fondamentaux de l'accessibilité - Vue d'ensemble"